src/main/java/ch/njol/skript/conditions/CondIsBaby.java

```diff
@@ -0,0 +1,38 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Ageable;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Is Baby")
+@Description("Checks whether or not a living entity is a baby.")
+@Examples({
+	"on drink:",
+		"\tevent-entity is a baby",
+		"\tkill event-entity"
+})
+@Since("INSERT VERSION")
+public class CondIsBaby extends PropertyCondition<LivingEntity> {
+
+	static {
+		register(CondIsBaby.class, "[a] (child|baby)", "livingentities");
```

Comment:
if me is baby

in seriousness, we may want to enforce a/an, but up to you.

Reply:
hahaha